src/plugins/saved_objects_management/public/lib/resolve_import_errors.ts

```diff
@@ -167,6 +168,7 @@ export async function resolveImportErrors({
   http,
   getConflictResolutions,
   state,
+  workspaces,
```

Comment:
Seems this attribute doesn't exists in the parameter type.

Reply:
Thanks, I've add that.